src/views/feedback/feedback.vue

```diff
@@ -7,130 +7,78 @@
             />
           </el-tab-pane>
           <el-tab-pane :label="`未查看(${unReadFeedBacks.total})`" :name="TAB.UNREAD">
             <!-- 未查看 -->
             <FeedBackTable
-              :tableData="unReadFeedBacks.records"
+              :tableData="unReadFeedBacks"
               @replyQuick="onReplyQuick"
               @viewDetail="onViewDetail"
+              @changePage="onChangeUnReadPage"
             />
           </el-tab-pane>
         </el-tabs>
       </el-card>
     </div>
     <!-- 对话框 -->
-    <FeedBackDiag ref="FeedBackDiagRef" :currentFeedBack="currentFeedBack" :replyMsg="replyMsg" />
-    <!-- 全部 pagination -->
-    <pagination
-      v-show="currentTab === TAB.ALL"
-      :total="allFeedBack.total"
-      :page="allFeedBack.current"
-      :limit="allFeedBack.size"
-      @pagination="onChangeAllPage"
-      class="pagination_feedBack"
-    />
-    <!-- 已读 pagination -->
-    <pagination
-      v-show="currentTab === TAB.READ"
-      :total="readFeedBacks.total"
-      :page="readFeedBacks.current"
-      :limit="readFeedBacks.size"
-      @pagination="onChangeReadPage"
-      class="pagination_feedBack"
-    />
-    <!-- 未读 pagination -->
-    <pagination
-      v-show="currentTab === TAB.UNREAD"
-      :total="unReadFeedBacks.total"
-      :page="unReadFeedBacks.current"
-      :limit="readFeedBacks.size"
-      @pagination="onChangeUnReadPage"
-      class="pagination_feedBack"
-    />
+    <FeedBackDialog ref="feedBackDiagRef" :currentFeedBack="currentFeedBack" />
   </div>
 </template>
-
 <script lang="ts" setup>
+// 回复用户
+
 import { ref, onMounted, computed } from 'vue'
 import { type TabsPaneContext } from 'element-plus'
 import { useFeedBackStore } from '@/store/modules/feedback'
-
 import { pageInfo } from '@/types/pageMessage'
-import { ReplyInfo } from '@/types/feedback'
 import { storeToRefs } from 'pinia'
 import FeedBackTable from './components/FeedBackTable.vue'
-import FeedBackDiag from './components/FeedBackDiag.vue'
-const enum TAB {
-  /**全部 */
-  ALL,
-  /**已查看 */
-  READ,
-  /**未查看 */
-  UNREAD
-}
-// 页数
-const PAGE_SIZE = 10
-
-// 偏移量
-const PAGE_OFFSET = 0
-
-// 回复标题
-const TITLE = '收到反馈'
-
-// 默认回复内容
-const DEFUALT_CONTENT = '感谢您的反馈！我们非常重视您提供的信息，我们会尽快处理并采取适当的措施。'
+import FeedBackDialog from './components/FeedBackDialog.vue'
+import { TAB } from '@/enums/Feedback'
+import { replyUser } from '@/api/feedback'
+import { replyMsg } from './constants'
 
 const feedBackStore = useFeedBackStore()
 const {
   getUnReadFeedbacks,
   getAllFeedBacks,
   getReadFeedbacks,
-  updateStateToRead,
-  replyToUser,
   updateStateReply,
+  updateStateToRead,
   refreshAllFeedBacks,
   refreshReadFeedbacks,
   refreshUnReadFeedbacks
 } = feedBackStore
 const { allFeedBack, readFeedBacks, unReadFeedBacks } = storeToRefs(feedBackStore)
 
 // 当前Tab状态
-const currentTab = ref(TAB.ALL)
+const currentTab = ref<TAB>(TAB.ALL)
 
 // 当前行的feedBackId
-const currentFeedbackId = ref()
+const currentFeedbackId = ref<number>()
```

Comment:
如果没有默认的 feedbackId，那默认应该给 null

```suggestion
const currentFeedbackId = ref<number | null>(null)
```